tests/tizen_capi/unittest_tizen_sensor.cpp

```diff
@@ -0,0 +1,451 @@
+/**
+ * @file	unittest_tizen_sensor.cpp
+ * @date	25 Nov 2019
+ * @brief	Unit test for NNStreamer's tensor-src-tizensensor.
+ * @see		https://github.com/nnsuite/nnstreamer
+ * @author      MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @bug         No known bugs
+ */
+
+#ifndef __TIZEN__
+/* These works only in Tizen */
+#error This unit test works only in Tizen. This needs Tizen Sensor Framework.
+#endif
+
+#include <glib.h>
+#include "dummy_sensor.h" /* Dummy Tizen Sensor Framework */
+#include <gtest/gtest.h>
+#include <gst/gst.h>
+#include <nnstreamer.h>
+#include <nnstreamer-capi-private.h>
+
+/**
+ * @brief Test pipeline creation of it
+ */
+TEST (tizensensor_as_source, virtual_sensor_create_01)
+{
+  gchar *pipeline;
+  sensor_event_s value;
+  sensor_h sensor;
+  GstElement *gstpipe;
+  GError *err = NULL;
+  int status = 0;
+
+  sensor_get_default_sensor (SENSOR_LIGHT, &sensor);
+
+  value.accuracy = 1;
+  value.timestamp = 0U;
+  value.value_count = 1;
+  value.values[0] = 0.01;
+  EXPECT_EQ (dummy_publish (sensor, value), 0);
+
+  /* Create a nnstreamer pipeline */
+  pipeline = g_strdup_printf ("tensor_src_tizensensor type=SENSOR_LIGHT sequence=0 num-buffers=3 ! fakesink");
+  gstpipe = gst_parse_launch (pipeline, &err);
+  if (gstpipe)
```

Comment:
to prevent svace issue, plz add gst_object_unref (gstpipe);

Reply:
Applied.